### PR TITLE
update parameter logic

### PR DESCRIPTION
**Pull request by adamdyszy**
refactor docker registry into registry and image, also change default parameter logic

### CODE REVIEW

File: pkg/apis/operator/v1alpha1/helper.go [hunk 1]
```diff
@@ -40,7 +40,14 @@ func (spec *IBMLicensingSpec) FillDefaultValues(isOpenshiftCluster bool) {
 		spec.IngressEnabled = &isNotOnOpenshiftCluster
 	}
 	if spec.ImageRegistry == "" {
-		spec.ImageRegistry = "quay.io/opencloudio/ibm-licensing-service"
+		spec.ImageRegistry = "hyc-cloud-private-integration-docker-local.artifactory.swg-devops.com/ibmcom/ibm-licensing-amd64"
```
Comment (ab-ibm) [inline pkg/apis/operator/v1alpha1/helper.go]: I think edge should be used not integration one

File: pkg/apis/operator/v1alpha1/helper.go [hunk 2]
```diff
@@ -40,7 +40,14 @@ func (spec *IBMLicensingSpec) FillDefaultValues(isOpenshiftCluster bool) {
 		spec.IngressEnabled = &isNotOnOpenshiftCluster
 	}
 	if spec.ImageRegistry == "" {
-		spec.ImageRegistry = "quay.io/opencloudio/ibm-licensing-service"
```
Comment (ab-ibm) [inline pkg/apis/operator/v1alpha1/helper.go]: Why we are not using opencloudio as default repo?

Comment (adamdyszy) [reply pkg/apis/operator/v1alpha1/helper.go]: fixed here: https://github.com/IBM/ibm-licensing-operator/pull/10/commits/387c3accdcd0309604af2460561daa802e6c1c78

**Comment by adamdyszy** [comment]
closing, related changes will be available at: https://github.com/IBM/ibm-licensing-operator/pull/10

**Pull request closed without merging**